pkg/chartverifier/pyxis/pyxis.go

```diff
@@ -92,7 +97,7 @@ func GetImageRegistries(repository string) ([]string, error) {
 	return registries, err
 }
 
-func IsImageInRegistry(repository string, version string, registry string) (bool, error) {
+func IsImageInRegistry(registry string, repository string, imageTag string, imageSha string) (bool, error) {
```

Comment:
Since we introduce in this PR new struct `ImageReference`, why do not we use it here as the function arg? For example:

```
func IsImageInRegistry(ref *ImageReference) (bool, error)
```

Reply:
I made this change but there was quite a knock on effect:

1. in ImageReference there is an array of registries. As a result had to move the loop logic from the caller of IsImageInRegistry to inside IsImageInRegistry

2. Pyxis code cannot import checks.ImageReference without creating a circular dependency so moved the structure to Pyxis.go and now checks.go uses Pyxis.ImageReference

3. The tests for  IsImageInRegistry had to be updated to provide an ImageReference instead of individual values.